src/EditorFeatures/Core/SymbolSearch/SymbolSearchUpdateEngineFactory.cs

```diff
@@ -100,10 +99,12 @@ public async ValueTask<ImmutableArray<PackageWithTypeResult>> FindPackagesWithTy
                 return result.HasValue ? result.Value : ImmutableArray<ReferenceAssemblyWithTypeResult>.Empty;
             }
 
-            public async ValueTask UpdateContinuouslyAsync(string sourceName, string localSettingsDirectory, CancellationToken cancellationToken)
+            public async ValueTask UpdateContinuouslyAsync(string sourceName, string localSettingsDirectory, ISymbolSearchLogService logService, CancellationToken cancellationToken)
             {
+                // logService parameter is ignore since it's already set on the connection as a callback
```

Comment:
```suggestion
                // logService parameter is ignore since it's already set on the connection as a callback
                _ = logService;
```